xbmc/utils/DatabaseUtils.h

```diff
@@ -126,6 +126,7 @@ typedef enum
   FieldSubtitleLanguage,
   FieldProductionCode,
   FieldTag,
+  FieldVideoVersion,
```

Comment:
Why is this field defined here?  The fields here are either used in sorting methods or smartplaylist generation, but this doesn't seem to be in either ?

Seems there is either more stuff to add (smartplaylists maybe?) or else its not needed.

Reply:
It's used in CVideoInfoTag::ToSortable

Reply:
Yeah, thats what I mean.  Just using the field there doesn't automatically make it sortable or usable in smart playlists or filters etc.  If thats the intention then there is more to do.  If not and it's just there for completeness then I doubt its needed.